content/docs/zero/import.md

```diff
@@ -0,0 +1,125 @@
+- prevents partial imports with quota checks.
+
+## Who is this for?
+
+The Core to Zero import feature is for users that want to use Pomerium Zero with their existing Pomerium Core configuration. Without the import capability, you’d have to manually configure each route, policy, certificate, and config setting in Pomerium Zero.
+
+This feature automates the process of importing your existing configuration so you don’t have to.
+
+## Core to Zero import definitions
+
+- **Entity:** any route, policy, certificate, or self-hosted domain defined in a Pomerium configuration.
+- **Replica:** a separate Pomerium configuration that shares a storage backend with other replicas running in the same deployment.
+- **Quota:** the amount of entities and replicas you can define in a cluster for free.
+
+## Supported settings
+
+Some settings available in Pomerium Core are not supported in Pomerium Zero. If unsupported settings are detected during an import, Pomerium will either remove them to complete the import or abort the import process.
+
+The table below lists unsupported settings, and which settings cause an import failure:
+
+| Unsupported config settings         | Fails import? |
+| ----------------------------------- | ------------- |
+| `authenticate_internal_service_url` | Yes           |
+| `authorize_internal_service_url`    | Yes           |
+| `databroker_internal_service_url`   | Yes           |
+| `derive_tls`                        | Yes           |
+| `audit_key`                         | No            |
+| `primary_color`                     | No            |
+| `secondary_color`                   | No            |
+| `darkmode_primary_color`            | No            |
+| `darkmode_secondary_color`          | No            |
+| `logo_url`                          | No            |
+| `favicon_url`                       | No            |
+| `error_message_first_paragraph`     | No            |
+| `use_proxy_protocol`                | No            |
+| `envoy_bind_config_freebind`        | No            |
+| `envoy_bind_config_source_address`  | No            |
+| `metrics_certificate`               | No            |
+| `metrics_client_ca`                 | No            |
+
+## Quotas
+
+When importing your Core configuration, the number of existing entities or replicas may exceed the personal account quotas in Pomerium Zero.
+
+To accommodate Core configurations with lots of entities or multiple replicas, we temporarily increase quotas during the import process. This exception prevents partial configuration imports.
+
+## How to import your Core configuration
+
+This guide shows you how to import an existing Core configuration to Pomerium Zero.
+
+### Before you start
+
+To complete this guide, you need a:
+
+- [Pomerium Zero](https://console.pomerium.app/create-account) account
+- [Pomerium Core](/docs/core/quickstart) configuration
+
+:::note
+
+This guide assumes Docker as the deployment environment.
```

Comment:
Let's call out that however you are using Core today, you can import the configuration into Zero.